addons/website_event_booth_sale/controllers/event_booth.py

```diff
@@ -20,20 +21,25 @@ def event_booth_main(self, event):
     @http.route()
     def event_booth_registration_confirm(self, event, booth_category_id, event_booth_ids, **kwargs):
```

Comment:
Could be nice to include a small docstring and explain why we don't call super.
(Because we need to go through the checkout process that will confirm the booth upon receiving the payment blablabla...).